multicluster/apis/multicluster/v1alpha1/resourceexportfilter_types.go

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
If we are not clear of the design yet, could we add it later?

Reply:
sure.

Reply:
this comment is not yet addressed

Reply:
@suwang48404 : @abhiraut is talking about my comment on whether we should remove the filter types for the PR, which are just dummy definition so far.